src/Main.ts

```diff
@@ -848,6 +863,7 @@ export class Main {
             checkToken: false,
         });
 
+        await this.applyBotProfile();
```

Comment:
I'd like this in a try catch block, where an error outputs an error message but does not cause the bridge to crash.